src/private/emscripten_api/emscripten_indoor_map_entity_information_api.js

```diff
@@ -133,7 +142,14 @@ function EmscriptenIndoorMapEntityInformationApi(emscriptenApiPointer, cwrap, ru
             var lng = positionLatLngArray[1];
             var position = L.latLng(lat, lng);
 
-            var entity = new indoorMapEntities.IndoorMapEntity(indoorMapEntityId, indoorMapEntityFloorId, position);
+            var Outlines = _GetOutlines(indoorMapEntityModelIds[entityIndex]);
+
+            if(Outlines === null)
```

Comment:
Is this correct? Surely we still want to create an IndoorMapEntity even if it doesn't have outline information available?